### DISCUSSION TITLE
Does react-hook-form support isValidating?

### DISCUSSION THREAD

**Discussion by anhquande** [react-hook-form/react-hook-form#2839] [Q&A] [answered]
Hello,

I switch from formik to react-hook-form. 
However, I could not find the isValidating in the formState ? 
Do I miss it or react-hook-form currently does not support it ?

My solution for now is: 

```
const {trigger} = formMethods;
const [formStatus, setFormStatus] = useState();

useEffect(()=>{
   if (allValuesUptodate){
       setFormStatus("validating ...");
       setTimeout(async () => {
          trigger();
          setFormStatus("");
       },0);
  }
}, [allValuesUptodate]);
```

**Comment by bluebill1049** [comment, marked as answer]
we don't have `isValidating` formState, you can request for a feature request and I will tag waiting for upvotes. for now, you will have to have that extra state live in your app.

**Reply by anhquande** [reply]
just did it https://github.com/react-hook-form/react-hook-form/discussions/2841
